packages/design-system/src/components/TextField/Mask.test.jsx

```diff
@@ -141,40 +141,51 @@ describe('Mask', function () {
   });
 
   describe('Currency', () => {
-    it('accepts already masked value', () => {
-      const data = render({ mask: 'currency' }, { value: '1,234.50' });
+    const testCurrency = (value, expected) => {
+      const data = render({ mask: 'currency' }, { value });
       const input = data.wrapper.find('input');
-
-      expect(input.prop('value')).toBe('1,234.50');
+      expect(input.prop('value')).toBe(expected);
+    };
```

Comment:
Ideally we would just be calling `toCurrency` directly instead of rendering the component each time, but I figured we shouldn't be exporting functions that aren't going to be used by the consumers of the DS.

Reply:
I think its actually ok to export `toCurrency` because we only export specific functions: https://github.com/CMSgov/design-system/blob/master/packages/design-system/src/components/index.js

Reply:
I actually like that it's testing the component as a whole. Maybe we can have both?